### DISCUSSION TITLE
希望在自定义主页中添加一个新的功能【启动当前选择的版本中指定的服务器】

### DISCUSSION THREAD

**Discussion by WSXYT** [Meloong-Git/PCL#3991] [提问] [answered]
注意是当前选择的版本！！！
在当前选择的版本启动指定的服务器！！！
在自定义主页！！！
指定是自定义主页指定的！！！
建议问完我再结束，我发起的两个都是理解错了……

**Comment by MoYuan-CN** [comment]
开 Issue
但是现在有计划做一个目前版本的占位符，如果有了可能可以用

**Comment by wuliaodexiaoluo** [comment]
如果你是指 Client 的 net.minecraft.server.Main 和 server 核心
抱歉，PCL 是 Minecraft 游戏启动器，不是 Minecraft 服务器启动器
如果是支持下载核心 见 #112 （#113）

**Reply by hejiehao** [reply]
他想说的应该是这个
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/994a8e25-e688-4f9c-b298-9260b03d3c2f)

**Reply by wuliaodexiaoluo** [reply]
```启动当前选择的版本中指定的服务器```
模糊性极强

**Comment by WForst-Breeze** [comment, marked as answer]
不要在 Discussion 中提交反馈，请移步 Issue 页面

**Comment by WorldHim** [comment]
哥们帮你交了
有补充可以在 #3997 (#4002) 下评论
